src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,32 @@ function createAllowPolicy(
   };
 }
 
+function getHeaderValue(
+  headers: Record<string, string> | undefined,
+  name: string
+): string | undefined {
+  if (!headers) {
+    return undefined;
+  }
+  const lcName = name.toLowerCase();
```

Comment:
Please rename `lcName`to `searchedHeaderName` or something like that.

Reply:
The whole function could be implemented more easily as follows:

```
function getHeaderValue(
  headers: Record<string, string> | undefined,
  name: string
): string | undefined {
  const searchedHeaderName = name.toLowerCase();

  return (
    headers &&
    Object.entries(headers).find(
      ([headerName]) => headerName.toLowerCase() === searchedHeaderName
    )?.[1]
  );
}
```

Reply:
Changed implementation of the function to the proposal above.